icons/webgpu/webgpu-original.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#0086e8" stroke="#0086e8" stroke-width=".193" d="M116.254 64 104.64 43.881h23.265z"/><path fill="#0093ff" stroke="#0093ff" stroke-width=".193" d="m116.254 23.762-11.615 20.12h23.265z"/><path fill="#0076cc" stroke="#0076cc" stroke-width=".193" d="M93.025 104.238 69.791 64h46.465Z"/><path fill="#0066b0" stroke="#0066b0" stroke-width=".193" d="M93.025 23.762 69.791 64h46.465Z"/><path fill="#005a9c" stroke="#005a9c" stroke-width=".193" d="M46.56 104.238.096 23.762h92.927Z"/></svg>
```

Comment:
Same issue here.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#0086e8" d="M104.473 43.785l11.78 20.408.085-.144 11.734-20.264h-23.6z"/><path fill="#0093ff" d="M116.254 23.568l-.084.145-11.697 20.266h23.6l-11.82-20.41z"/><path fill="#0076cc" d="M69.623 63.904l23.402 40.528.084-.145 23.315-40.383H69.623z"/><path fill="#0066b0" d="M93.025 23.568l-.084.145-23.318 40.383h46.8L93.026 23.568z"/><path fill="#005a9c" d="M-.07 23.666l46.63 80.766.085-.145 46.544-80.621H-.07z"/></svg>

```

Reply:
Can't believe I completely missed these.
Thanks for the suggestions @lunatic-fox